My Series List/My Series List/View/DetailsView/DetailsViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DetailsViewController.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 18/11/2021.
+//
+
+import UIKit
+
+class DetailsViewController: UIViewController, DetailsViewDelegate {
+
+    let detailsViewPresenter = DetailsViewPresenter()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        detailsViewPresenter.setDetailsViewDelegate(self)
+    }
+}
+
+extension DetailsViewController {
```

Comment:
It is this extension that implements methods from DetailsViewDelegate. This extension should adopt DetailsViewDelegate